lib/libvcc/vcc_expr.c

```diff
@@ -497,7 +497,6 @@ vcc_func(struct vcc *tl, struct expr **e, const void *priv,
 		if (!memcmp(vvp->value, "PRIV_", 5)) {
 			fa->result = vcc_priv_arg(tl, vvp->value,
 			    sym->name, sym->vmod);
-			fa->name = "";
```

Comment:
* either the function argument is NULL or properly named, we remove the case where it also can be the empty string in order to avoid additional tests and ambiguity